src/PrestaShopBundle/Resources/config/services/core/grid/grid_definition_factory.yml

```
@@ -386,3 +386,8 @@ services:
     parent: 'prestashop.core.grid.definition.factory.abstract_grid_definition'
     public: false
     autowire: true
+
+  prestashop.core.grid.definition.factory.alias:
+    class: 'PrestaShop\PrestaShop\Core\Grid\Definition\Factory\AliasGridDefinitionFactory'
```

ths should be autowirable
```suggestion
  PrestaShop\PrestaShop\Core\Grid\Definition\Factory\AliasGridDefinitionFactory:
    autowire: true
```